src/Exception/SvgStructureException.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Exception;
+
+use DOMElement;
```

Comment:
I don't think we are using `DomElement` in this class.

Reply:
Removed. I wonder if one day we should switch to the Wikimedia phpcs rules (they'd catch this sort of thing).